tests/PHPUnit/Model/KataTest.php

```diff
@@ -43,4 +45,14 @@ public function test_start_should_return_started_kata()
         $this->kata->addStep($this->getMock(Step::INTERFACE_NAME));
         $this->assertInstanceOf(StartedKata::CLASS_NAME, $this->kata->start());
     }
+
+    public function test_should_contain_objectives()
+    {
+        $this->assertAttributeCount(0, 'objectives', $this->kata);
+        $this->kata->addObjective(new NullObjective());
+        $this->assertAttributeCount(1, 'objectives', $this->kata);
+        $this->kata->addObjective(new NullObjective());
+        $this->kata->addObjective(new NullObjective());
+        $this->assertAttributeCount(3, 'objectives', $this->kata);
```

Comment:
Since i don't know how the objective will work on the valudate method, i just wanted to make sur the elements were added correctly.